### ISSUE TITLE
The accuracy results differ greatly compared to those in the paper

### ISSUE THREAD

**Issue by Yifan-Lu-47** [ds4dm/learn2branch-ecole#5]
![results](https://github.com/user-attachments/assets/fdac48b1-2906-4bb4-bc55-dbbc956309d1)
I found that the accuracy results are significantly lower than those from the TensorFlow version. Taking the SetCover problem as an example, the accuracy was originally above 60%, but with the Ecole version, the result is only around 9%, as shown in the figure. Could you clarify what the typical accuracy range is for this version?

**Comment by Zjz12138** [comment]
True_scores has a large number of 'nan' causing errors in calculating 'acc'.

You can change the following code:
            true_scores = pad_tensor(batch.candidate_scores, batch.nb_candidates)
            true_bestscore = true_scores.max(dim=-1, keepdims=True).values
to：
            true_scores = pad_tensor(batch.candidate_scores, batch.nb_candidates)
            true_scores = torch.where(torch.isnan(true_scores), torch.tensor(-float('inf'), device=true_scores.device), true_scores)
            true_bestscore = true_scores.max(dim=-1, keepdims=True).values

**Comment by Zjz12138** [comment]
delete ’**pseudo_candidates=True**‘

**Comment by Debellert1** [comment]
I also find this.

**Comment by skanda-vyas-srinivasan** [comment]
I keep running into the same issue too. It reaches some optima during epoch 3-4 with accuracy ~ 0.07.
